textBasedAdventure.js

```diff
@@ -1,5 +1,245 @@
-const readline1 = require('readline-sync')
+const readline1 = require("readline-sync");
+console.clear()
 
-let nameInput = readline.question("Enter your name: ")
+let nameInput = readline1.question("Enter your name: ");
+console.clear()
 
-console.log(`Hello ${nameInput}!  Welcome to my game.`)
+console.log(`Hello ${nameInput}!  Welcome to Will you sleep? `);
+
+console.log(
+  "\nYou will be given different choices on whether you want to remain asleep or stay awake \n"
+);
+
+console.log("Its 5am and a explosion wakes you up!\n");
+
+let startGame = decisions();
+let endGame = process.exit();
+
+function decisions() {
+  let person = readline1.question(
+    "Do you A: roll over? Or B: go check it out?\n"
+  );
+  if (person === "B".trim().toLowerCase()) {
```

Comment:
Consider why you're using .trim() and .toLowerCase() on the letter "B"? It would make more sense to use it on the person variable. Understanding why you're using it will help to know where to use it.